nss-evening-cohort-14/renaissance-fair-dashboard-renaissance-fair
# Update and delete events along with show, staff, food and souvenir selections.

## Description
Events can be updated and deleted along with the data associations of shows, staff, food and souvenirs tied to an event.  The user can also add the counts for food and souvenir Items when creating or updating events.  

The input form allows for easy selection using checkboxes, and integer input for items requiring a count.

Events are displayed on the single event details view page along with the selected shows, staff, souvenirs and food.

## Related Issue
#118
#119
#120
#121
#139

## Motivation and Context
This allow for full creation, read, update and delete of events with associated data.

## How Can This Be Tested?
Fetch jm-crud-with-counts. 
Update an event by clicking the edit button.  Select your changes and submit. View your changes in the single event view. 
Delete an event, By viewing the firebase realtime database, it is possible to view the associated join tables being deleted when deleting an event.
Additionally, create a new event and check that your selections appear in the single event view.

## Types of changes
